.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+## Summary
```

Comment:
I think this directory should not be adding to the repository. Add the directory in .gitignore

Reply:
According to [Github documentation](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository) we can use .github directory if we want to hide the directory for this template. But, for some reason it is not working right now. I will change for **docs** directory